crest/Assets/Crest/Crest/Scripts/Interaction/SphereWaterInteraction.cs

```diff
@@ -236,7 +260,7 @@ public void Draw(LodDataMgr lodData, CommandBuffer buf, float weight, int isTran
             if (_debugSubsteps)
             {
                 var col = 0.7f * (Time.frameCount % 2 == 1 ? Color.green : Color.red);
-                var pos = transform.position + /*(fixup ? 1f : 0f) **/ -_velocity * timeBeforeCurrentTime;
+                var pos = transform.position + /*(fixup ? 1f : 0f) **/ -_velocity * (timeBeforeCurrentTime - _velocityOffset);
```

Comment:
Does something need to happen with this?

Reply:
cleaned up now